### PR TITLE
Feat color tracking from camera

### PR DESCRIPTION
**Pull request by mrzilvis**
Review my pull request

### CODE REVIEW

File: TOPBanga/VideoFromFile.cs [hunk 1]
```diff
@@ -84,5 +88,24 @@ private void DetectionButton_Click(object sender, EventArgs e)
             });
             this.videoTickTimer.Start();
         }
+
+        private void switchCam_Click(object sender, EventArgs e)
+        {
+            this.videoTickTimer.Interval = webcamInterval; //magic number
```
Comment (pauliuskalv) [inline TOPBanga/VideoFromFile.cs]: Remove unnecessary comment

File: TOPBanga/VideoFromFile.cs [hunk 2]
```diff
@@ -9,14 +9,15 @@ namespace TOPBanga
...
+        private const int webcamInterval = 80;
         private IDetector detector;
         private VideoCapture video;
         private Mat currentFrame;
         private System.Timers.Timer videoTickTimer;
         private bool videoLoaded;
-
-
+        private VideoCapture webcam;
+        private bool webcamOn;
```
Comment (pauliuskalv) [inline TOPBanga/VideoFromFile.cs]: This variable is unneeded if we check whether the webcam is on by checking if it is null like in line 100

File: TOPBanga/VideoFromFile.cs [hunk 3]
```diff
@@ -63,13 +65,13 @@ public void setDeltaText(String text)
...
-            if (!videoLoaded)
-                return;
             this.videoTickTimer.Stop();
             this.videoTickTimer = new System.Timers.Timer();
-            this.videoTickTimer.Interval = 30;
             this.videoTickTimer.Elapsed += new ElapsedEventHandler(delegate (object o, ElapsedEventArgs args) {
-                this.currentFrame = this.video.QueryFrame();
+                if (this.videoLoaded)
+                    this.currentFrame = this.video.QueryFrame();
+                else if (webcam == null)
```
Comment (solarbeam) [inline TOPBanga/VideoFromFile.cs]: Are you sure that the condition should be webcam == null ? You're using the webcam variable after checking that it doesn't exist.

File: TOPBanga/VideoFromFile.cs [hunk 4]
```diff
@@ -84,5 +86,23 @@ private void DetectionButton_Click(object sender, EventArgs e)
...
+
+        private void switchCam_Click(object sender, EventArgs e)
+        {
+            this.videoTickTimer.Interval = webcamInterval;
+            if (this.videoLoaded)
+            {
+                this.videoTickTimer.Stop();
+                this.videoLoaded = false;
+            }
+            if (webcam == null)
```
Comment (solarbeam) [inline TOPBanga/VideoFromFile.cs]: Shouldn't webcam be this.webcam?

**Pull request merged**